rpcclient/infrastructure.go

```diff
@@ -412,17 +411,27 @@ out:
 			break out
 		default:
 		}
-
-		_, msg, err := c.wsConn.ReadMessage()
-		if err != nil {
-			// Log the error if it's not due to disconnecting.
-			if c.shouldLogReadError(err) {
-				log.Errorf("Websocket receive error from "+
-					"%s: %v", c.config.Host, err)
+		//check for all wsConns with balancer
```

Comment:
Please follow the conventions of the other comments in the code.  e.g. full sentences with a period and a space after the double slashes.

This is the true throughout the entire PR, but I'll refrain from commenting on each instance and just mention it here.